templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/fragment_statement.html.twig

```diff
@@ -52,23 +52,25 @@
                 :init-priority-areas="JSON.parse('{{ selectedPriorityAreas|default([])|json_encode|e('js', 'utf-8') }}')"
                 init-fragment-text="{{ fragment.text|default|wysiwyg }}"
                 procedure-id="{{ procedure }}"
+                statement-text="{{ templateVars.statement.text|default|escape|wysiwyg(hasPermission('feature_obscure_text') ? 'dp-obscure' : []) }}"
                 inline-template>
                 <div>
                     <p class="u-mt-0_5">{{ "text.statement.fragment"|trans|wysiwyg }}</p>
 
                     {# statement text #}
                     <p class="weight--bold">
                         {{ "statement"|trans }}
-                        <dp-copy-paste-button
-                            button-label="statement.copy"
-                            class="float--right"
-                            copy-content="{{ templateVars.statement.text|default|escape|wysiwyg(hasPermission('feature_obscure_text') ? 'dp-obscure' : []) }}"
+                        <dp-button
+                            class="float--right btn--blank color-main font-size-medium"
```

Comment:
The color classes should be obsolete when using the subtle button variant. And, why must the font size set to the default here, again? Is it to change the icon size? 

```suggestion
                            class="float--right"
```

Reply:
The `btn` class (added in DpButton) sets the `font-size` to 14px, which seems very small?
I removed the other classes in 1a17c3d6bad190833d018e2aa4b3c54d7ad31825